wp-content/themes/goonj-crm/functions.php

```diff
@@ -88,3 +88,57 @@ function goonj_custom_password_reset_redirection( $errors, $user ) {
         exit;
     }
 }
+
+add_action('template_redirect', 'goonj_user_identification');
+function goonj_user_identification(){
+    if (is_page('user-identification-form')) {
+        get_template_part('templates/user-identification');
+    }
+}
+
+add_action('wp', 'goonj_handle_user_identification_form');
+function goonj_handle_user_identification_form() {
+    if ($_SERVER["REQUEST_METHOD"] == "POST" && isset($_POST['action']) && $_POST['action'] == 'user-identification') {
+        // Retrieve the email and phone number from the POST data
+        $email = isset($_POST['email']) ? $_POST['email'] : '';
```

Comment:
These can be simplied using the [null-coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op)

```suggestion
        $email = $_POST['email'] ?? '';
```